views/services/bookSecureMove.njk

```diff
@@ -5,9 +5,31 @@
 {% block content %}
   <h1 class="govuk-heading-l">{{title}}</h1>
 
-  <p>This service allows you to schedule secure movement for prisoners in custody, via approved transport suppliers, between locations across England and Wales.</p>
+  <p>This service allows you to schedule secure movement for someone in custody between locations across England and Wales.</p>
 
-  <p>If you have any problems with this service, contact the helpdesk by calling 0800 917 5148, or #6598 from inside a prison.</p>
+  <p>This is only for the movement of prisoners using the Prisoner Escort and Custodial Services (PECS) suppliers - these are:</p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>GEOAmey in Wales and in the north of England</li>
+    <li>Serco in the south of England</li>
+  </ul>
+
+  <p>The service cannot be used for movements completed by any other escorts except GEOAmey and Serco.</p>
+
+  <p>Once a move has been scheduled, the service provides details to the suppliers and the receiving location of the person being moved.</p>
+
+  <p>The service is also used for the completing a digital Person Escort Record (dPER), which provides relevant details to the escort suppliers and the receiving location. Details can include:</p>
```

Comment:
I think there is a typo here ("for the completing a"), but it is in the ticket so maybe it makes sense